README.md

```diff
@@ -108,5 +108,9 @@ $ polymer serve
 
 Open [localhost:8000](http://localhost:8000/)
 
+### Commit message formats
```

Comment:
This is not needed here. 

A full spec for how to manage feature branches and pull requests can be found here. 
https://auro.alaskaair.com/getting-started/developers/contributing

Please remove this commit.

Reply:
Could that documentation then include the notice that their local commits on their machine won't go through unless they also conform to the standard?